app/classifier/drawing-tools/ellipse.cjsx

```diff
@@ -53,6 +56,13 @@ module.exports = React.createClass
     x: @props.mark.rx * Math.cos theta
     y: -1 * @props.mark.ry * Math.sin theta
 
+  dragEnd: (target) ->
+    @context.geordi?.logEvent type: 'drag-ellipse'
+    deleteIfOutOfBounds target
+
+  logResize: ->
+    @context.geordi?.logEvent type: 'resize-ellipse'
+
```

Comment:
Geordi can handle it in terms of accepting the data - we've done load testing for that. It's more of an ops issue in terms of how big do we want the database to get. My gut feeling is we shouldn't capture drag/resize events except for specific experiments/cases.